### ISSUE TITLE
Unable to install on Android Firefox 

### ISSUE THREAD

**Issue by kuro68k** [uBlockOrigin/uBOL-home#221]
Since leaving the Mozilla add-ons site and moving to GitHub distribution, it's become impossible to install the latest version on Android in Firefox.

Firefox broke opening local xpi files and local HTML files at some point. There's no way to install uBOL now.

**Comment by krystian3w** [comment]
Possible for Android 10+ and Firefox 123+ (for Android 9 and lower needed is Firefox 120-122 and later upgrade to 131+):

settings :arrow: about program /about Firefox :arrow: tap 5 times at  Firefox logo :arrow: back to main Settings and the list should include:

„Install extension from file”.

**Comment by kuro68k** [comment]
Thank you krystian3w. I installed it that way and it did indeed work, although it is showing as version 20204.9.1.1266, rather than the 2024.9.22.986 that the file is labelled as.

I would say that this isn't an ideal solution for most users though.

**Comment by krystian3w** [comment]
It remains to be seen whether Moz://a plans to simplify installation from external domains even more, as on PCs and laptops (but it may take them dozen years, as everyone will sooner switch from Android 9 to 10-26).

**Comment by yokoffing** [comment]
uBOL on Firefox desktop also seems to be stuck on `2024.9.1.1266`. I tried to force update from the last [Firefox release on GitHub](https://github.com/uBlockOrigin/uBOL-home/releases/tag/uBOLite_2024.9.22.986) with no luck.

**Comment by krystian3w** [comment]
There is no newer file than the September one in the releases.

https://github.com/uBlockOrigin/uBOL-home/blob/main/dist/firefox/updates.json

**Comment by yokoffing** [comment]
> There is no newer file than the September one in the releases.

Correct. Perhaps I should've posed a question: Does Firefox-uBOL not need updating as often as Chrome-uBOL?

**Comment by garry-ut99** [comment]
> kuro68k : although it is showing as version `20204.9.1.1266`, rather than the `2024.9.22.986` that the file is labelled as.

> yokoffing: uBOL on Firefox desktop also seems to be stuck on `2024.9.1.1266`

It's just a version numbering typo on uBlock's part in `2024.9.22.986` release in `manifest.json` file, `2024.9.22.986` is `2024.9.22.986`, and not `20204.9.1.1266`, so nothing to worry other than wrong version number.

<hr>

> yokoffing: Correct. Perhaps I should've posed a question: Does Firefox-uBOL not need updating as often as Chrome-uBOL?

See:
- https://github.com/uBlockOrigin/uBOL-home/issues/197
- https://github.com/uBlockOrigin/uBOL-home/discussions/231

**Comment by yokoffing** [comment]
> https://github.com/uBlockOrigin/uBOL-home/discussions/231#discussioncomment-10961391

Thank you.

**Comment by krystian3w** [comment]
Someone plans as https://github.com/rav3ndust/happyfox

But 0 match with https://www.google.com/search?q=site%3Aaddons.mozilla.org+%22happyfox%22 (so will be private or hosted on outside of webstore).

**Comment by gwarser** [comment]
Refined Github feature - https://github.com/refined-github/refined-github/blob/main/source/features/comments-time-machine-links.tsx
